orientdb/src/test/java/org/hibernate/datastore/ogm/orientdb/OrientDBSimpleTest.java

```diff
@@ -0,0 +1,280 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.datastore.ogm.orientdb;
+
+import com.orientechnologies.orient.core.id.ORecordId;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.FlushModeType;
+import javax.persistence.Persistence;
+import javax.persistence.Query;
+import org.apache.log4j.BasicConfigurator;
+import org.hibernate.datastore.ogm.orientdb.jpa.Customer;
+import org.hibernate.datastore.ogm.orientdb.jpa.Pizza;
+import org.hibernate.datastore.ogm.orientdb.util.MemoryDBUtil;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+import org.junit.FixMethodOrder;
+import org.junit.runners.MethodSorters;
+
+/**
+ * Test checks CRUD for simple entities (without links with other entities)
+ * 
+ * @author Sergey Chernolyas (sergey.chernolyas@gmail.com)
+ */
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class OrientDBSimpleTest {
+
+	private static final Logger LOG = Logger.getLogger( OrientDBSimpleTest.class.getName() );
+	private static EntityManager em;
+	private static EntityManagerFactory emf;
+
+	@BeforeClass
+	public static void setUpClass() {
+		LOG.log( Level.INFO, "start" );
+		// MemoryDBUtil.prepareDb("remote:localhost/pizza");
+		MemoryDBUtil.createDbFactory( "memory:test" );
+		BasicConfigurator.configure();
+		emf = Persistence.createEntityManagerFactory( "hibernateOgmJpaUnit" );
+		em = emf.createEntityManager();
+		em.setFlushMode( FlushModeType.COMMIT );
+
+	}
+
+	@AfterClass
+	public static void tearDownClass() {
+		LOG.log( Level.INFO, "start" );
+		if ( em != null ) {
+			em.close();
+			emf.close();
+
+		}
+		MemoryDBUtil.getOrientGraphFactory().close();
```

Comment:
By the way, if your test class extends `OgmTestCase`  and the datastore provider is implemented correctly, the SessionFactory is created automagically and the database started and closed.